Library/Homebrew/cask/lib/hbc/cli/cleanup.rb

```diff
@@ -75,14 +75,18 @@ def delete_paths(paths)
         paths.each do |item|
```

Comment:
Yes, basically it just loops through each file in `HOMEBREW_LOCK_DIR`, tries to lock it, and removes it if it can lock it, so the suffix doesn't matter.